File: Backend/TravellifeChaser/Models/AirlinesSystem/Many-To-ManyEntities/FlightAirport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TravellifeChaser.Models
{
    public class FlightAirport
    {
        public int FlightId { get; set; }
        public Flight Flight { get; set; }

        public int AirportId { get; set; }
        public Airport Airport { get; set; }
    }
}

File: AvioAndRentACarSystem/src/app/AirlineModel/flightAirport.ts
import { Flight } from './flight';

export class FlightAirport {
    flightId: number;
    flight: Flight;

    airportId: number;
    airport: number;

    constructor(flightId: number, airportId: number) {
        this.flightId = flightId;
        this.airportId = airportId;
    }
}